src/wallet.js

```diff
@@ -153,7 +153,7 @@ const wallet = {
     let initialAccessData;
     let privkey;
     if (xpriv.depth === 0) {
-      privkey = xpriv.derive(`m/44'/${HATHOR_BIP44_CODE}'/0'/0`);
+      privkey = xpriv.deriveChild(`m/44'/${HATHOR_BIP44_CODE}'/0'/0`);
```

Comment:
In a separate PR, add a test that create a wallet with given seeds and check their first 5 addresses. This test must pass without these changes and with these changes. It will guarantee that the address generation remains the same.